love/src/components/EventLog/EventLog.container.jsx

```diff
@@ -28,6 +28,12 @@ export const schema = {
       isPrivate: false,
       default: 1,
     },
+    hierarchy: {
+      type: 'object',
```

Comment:
I was thinking that we could have a `list` instead of an `object`. The `CSCSummaryHierarchy` was defined as an object because the CSCSummary view needed to define different sections of content. In this case we only want to have a list of CSCs to listen, that's why I suggest to use a list instead. Take a look at: https://github.com/lsst-ts/LOVE-frontend/pull/325/files#diff-57e970eb4eaa1eec9e9e262e424ecf02556a31c859067b25699664f7f7db4754R18

Reply:
finally used of list of CSC with attributes name and salindex